### PR TITLE
Utility functions to handle subSystems

### PR DESCRIPTION
**Pull request by tpfau**
This PR provides a set of utility functions to handle subSystems. As subSystems is now a cell arrays of cell arrays, manual manipulation is no longer straight forward.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/reconstruction/refinement/setRxnSubSystems.m [hunk 1]
```diff
@@ -0,0 +1,55 @@
+function [model]  = setRxnSubSystems(model,reactions,subSystems)
+% Adds the subSystem to the specified reaction
```
Comment (laurentheirendt) [inline src/reconstruction/refinement/setRxnSubSystems.m]: Is this description correct? It is the same as in `removeSubSystemsFromRxns.m`?

**Comment by tpfau** [comment]
@laurentheirendt  Any additional changes?

**Comment by rmtfleming** [comment]
Can be merged.

**Comment by laurentheirendt** [comment]
Thanks @tpfau 🌵

**Pull request merged**